src/plugins/data_table/filter_view.js

```diff
@@ -12,19 +12,19 @@ var DataTableFilter = (function(options) {
 
     menuTemplate: _.template(''), // to be overridden by subclasses
     parentMenuTemplate: _.template('\
-      <div class="filterMenu <%= filterMenuClass %>">\
-        <%= menuTemplate %>\
-        <button class="btn btn-sm btn-filter" name="button" type="submit" title="">Apply</button>\
-        <button class="btn btn-primary btn-sm btn-clear pull-right" name="button" type="submit" title="">Clear</button>\
-      </div>\
+     <div class="filterMenu <%= filterMenuClass %>"> \
+       <% if (enabled) { %> \
+         <%= menuTemplate %> \
+         <button class="btn btn-sm btn-filter" name="button" type="submit" title="">Apply</button>\
+         <button class="btn btn-primary btn-sm btn-clear pull-right" name="button" type="submit" title="">Clear</button>\
+       <% } else { %> \
+         <span class="error-text"> \
```

Comment:
because there are tests that rely on this element and we don't want to couple tests (esp cross-repo tests) to CSS classes that are providing style... this should also have a `data-mount="error-message"` attribute on it. If it's not too late can you add that please?